### DISCUSSION TITLE
didactisch-concept

### DISCUSSION THREAD

**Discussion by fjkresin** [fdnd/docs.fdnd.nl#280] [Reflectie]
"Dit blijkt uit prachtige NSE resultaten, schooljaar 2023/24 kreeg FDND het predicaat Topopleiding en voor 2024/25 ligt dit opnieuw in de lijn der verwachting." - ik zou dat laatste altijd afzwakken; je weet nooit of het weer lukt (al hoop & gun ik het jullie natuurlijk van harte)

**Comment by ju5tu5** [comment]
:D je hebt gelijk.. ik heb de tekst herschreven naar een wens..
